org.eclipse.m2e.launching/META-INF/MANIFEST.MF

```diff
@@ -18,7 +18,8 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.m2e.maven.runtime;bundle-version="[1.16.0,2.0.0)",
  org.eclipse.m2e.core;bundle-version="[1.16.0,2.0.0)",
  org.eclipse.m2e.core.ui;bundle-version="[1.16.0,2.0.0)",
- org.eclipse.m2e.workspace.cli;bundle-version="0.1.0"
+ org.eclipse.m2e.workspace.cli;bundle-version="0.1.0",
+ org.eclipse.pde.core
```

Comment:
Please don't couple m2e launching with PDE. If you want to add PDE specific behavior contributed to m2e launching, please introduce an extension point.

Reply:
Good catch! This would require to install PDE in all cases!

Reply:
With the approach suggested above coupling to PDE should not be necessary anymore.